fixit/cli/add_new_rule.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Usage:
+#
+#   $ python -m fixit.cli.add_new_rule --help
+#   $ python -m fixit.cli.add_new_rule
+#   $ python -m fixit.cli.add_new_rule --path fixit/rules/new_rule.py
+
+import argparse
+from pathlib import Path
+
+from libcst.codemod._cli import invoke_formatter
+
+from fixit.common.config import get_lint_config
+
+
+_LICENCE = """# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+"""
+
+_IMPORTS = """
+import libcst as cst
+import libcst.matchers as m
+
+from fixit import CstLintRule, InvalidTestCase as Invalid, ValidTestCase as Valid
+
+"""
+
+_TO_DOS = """
+\"""
+This is a model rule file for adding a new rule to fixit module
+\"""
+
+"""
+
+_RULE_CLASS = """
+class Rule(CstLintRule):
+    \"""
+    docstring or new_rule description
+    \"""
+
+    MESSAGE = "Enter rule description message"
+
+    VALID = [Valid("'example'")]
+
+    INVALID = [Invalid("'example'")]
+"""
+
+
+def is_path_exists(path: str) -> Path:
+    """Check for valid path, if yes, return `Path` else raise `Error` """
+    filepath = Path(path)
+    if filepath.exists():
+        raise FileNotFoundError(f"{filepath} already exists")
```

Comment:
FileExistsError is better aligned with the error message.
```suggestion
        raise FileExistsError(f"{filepath} already exists")
```